betty/wiki/client.py

```diff
@@ -57,6 +58,18 @@ class Image:
     title: str
     wikimedia_commons_url: str
     name: str
+    image_copyright: Copyright | None
```

Comment:
The class is called "image" already, so we can simply call the attribute "copyright".